pkg/devfile/parser/context/content.go

```diff
@@ -55,17 +55,16 @@ func (d *DevfileCtx) SetDevfileContent() error {
 		return errors.Wrapf(err, "failed to read devfile from path '%s'", d.absPath)
 	}
 
-	// If YAML file convert it to JSON
-	d.rawContent, err = YAMLToJSON(data)
-	if err != nil {
+	// set devfile content
+	if err := d.SetDevfileContentFromBytes(data); err != nil {
```

Comment:
Could be simplified to simply `return d.SetDevfileContentFromBytes(data)`

Reply:
Done